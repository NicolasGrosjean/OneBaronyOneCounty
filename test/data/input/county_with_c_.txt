	c_coptic_papacy = {
		b_coptic_papacy = {
			province = 42
			color = { 217 155 177 }
			color2 = { 220 220 0 }
		}
	}
