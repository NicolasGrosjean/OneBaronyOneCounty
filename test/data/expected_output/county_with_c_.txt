	c_coptic_papacy = {
		color = { 217 155 177 }
		color2 = { 220 220 0 }

		b_coptic_papacy = {
			province = 42
			color = { 217 155 177 }
			color2 = { 220 220 0 }
		}
	}
